src/midi/midi_mt32.cpp

```diff
@@ -61,12 +64,96 @@ MidiHandler_mt32 mt32_instance;
 static void init_mt32_dosbox_settings(Section_prop &sec_prop)
 {
 	constexpr auto when_idle = Property::Changeable::WhenIdle;
-	auto *str_prop = sec_prop.Add_string("romdir", when_idle, "");
+
+	const char *models[] = {"auto", "cm32l", "mt32", 0};
+	auto *str_prop = sec_prop.Add_string("model", when_idle, "auto");
+	str_prop->Set_values(models);
+	str_prop->Set_help(
+	        "Model of synthesizer to use. The default (auto) prefers CM-32L\n"
+	        "if both sets of ROMs are provided. For early Sierra games and Dune 2\n"
+	        "it's recommended to use 'mt32', while newer games typically made\n"
+	        "use of the CM-32L's extra sound effects (use 'auto' or 'cm32l')");
+
+	str_prop = sec_prop.Add_string("romdir", when_idle, "");
 	str_prop->Set_help(
-	        "The directory holding the required MT-32 Control and PCM ROMs.\n"
-	        "The ROM files should be named as follows:\n"
-	        "  MT32_CONTROL.ROM or CM32L_CONTROL.ROM - control ROM file.\n"
-	        "  MT32_PCM.ROM or CM32L_PCM.ROM - PCM ROM file.");
+	        "The directory holding the required MT-32 and/or CM-32L ROMs\n"
+	        "named as follows:\n"
+	        "  MT32_CONTROL.ROM or CM32L_CONTROL.ROM - control ROM files(s).\n"
+	        "  MT32_PCM.ROM or CM32L_PCM.ROM - PCM ROM file(s).\n"
+	        "The directory can be absolute or relative, or leave it blank to\n"
+	        "use the 'mt32-rom-data' directory in your DOSBox configuration\n"
+	        "directory, followed by checking other common system locations.");
+}
+
+#if defined(WIN32)
+
+static std::list<std::string> get_rom_dirs(const std::string &preferred_dir)
+{
+	return {
+	        preferred_dir,
+	        CROSS_GetPlatformConfigDir() + "mt32-rom-data\\",
```

Comment:
I was thinking about more generic `romdir` (in case we'll ever support loading any other hardware ROM files). Another options would be `mt32-roms` (as `data` suggests we have some additional data in there).

Reply:
Sounds good; I also like 'mt32-roms' and it makes more sense. Will add it as the new default ahead of the existing ones.

The basis of using "`mt32-rom-data` just comes from it being used by Munt and its frontend for the past ~15 years. So probably need to retain that as the last set of search paths.